Source/FieldSolver/SpectralSolver/SpectralFieldData.H

```diff
@@ -33,11 +33,13 @@ struct SpectralAvgFieldIndex {
   // n_fields is automatically the total number of fields
 };
 
-/* Index for the PML fields, when stored in spectral space */
+/** Index for the PML fields, when stored in spectral space,
+ *  (n_fields is automatically set to the total number of fields)
+ *  TODO How to include the diagonal components only when needed?
+ */
 struct SpectralPMLIndex {
-  enum { Exy=0, Exz, Eyx, Eyz, Ezx, Ezy,
-           Bxy, Bxz, Byx, Byz, Bzx, Bzy, n_fields };
-  // n_fields is automatically the total number of fields
+  enum {Exx=0, Exy, Exz, Eyx, Eyy, Eyz, Ezx, Ezy, Ezz,
+        Bxx  , Bxy, Bxz, Byx, Byy, Byz, Bzx, Bzy, Bzz, Fx, Fy, Fz, Gx, Gy, Gz, n_fields};
 };
```

Comment:
Does it matter if there are redundant enumerators that are not always used (for example, the diagonal enumerators and the ones for F and G)?